package.json

```diff
@@ -10,6 +10,7 @@
     "setup": "yarn install && yarn setup:postinstall",
     "setup:postinstall": "yarn patch-package && yarn allow-scripts",
     "start": "yarn build:dev dev",
+    "start:mv3": "ENABLE_MV3=true yarn build:dev dev",
```

Comment:
here you are setting `ENABLE_MV3=true` should it be `ENABLE_MV3='1'` to match what is expected in `.metamaskrc`. I tried this btw and it seems that setting it to `'1'` makes it work as expected.

Reply:
@adonesky1 : `ENABLE_MV3=true` also works well for me.